dockers/docker-database/database_global.json.j2

```diff
@@ -15,7 +21,22 @@
         },
 {% endif %}
 {% endfor %}
+{% endif %}
+
+{% if dpu_cnt > 0 %}
+{% for dpu in range(dpu_cnt) %}
+        {
+            "database_type" : "dpudb",
+            "include" : "../../redisdpu{{dpu}}/sonic-db/database_config.json"
```

Comment:
nit: i wonder if we could use "redis-dpu{{dpu}}"

Reply:
and we can do it in another change, since this change is already approved.

Reply:
It's not easy. Because the database folder consists of the prefix "redis" and suffix "$DEV" without the dash "-". This convention has been used in the multi-ASIC scenario.